tests/unit/ResultsItemDetails.spec.js

```diff
@@ -0,0 +1,155 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import ResultsItemDetails from '@/components/ResultsItemDetails.vue';
+import Vuex from 'vuex';
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+
+describe('ResultsItemDetails', () => {
+  let state, store, mutations, computed, $route;
+
```

Comment:
Nice! I always forget that you can do multiple declarations in one line. So fresh and so clean.